mod/metrics/set/ltype_rtype/metric/wikirate_company.rb

```diff
@@ -1,4 +1,6 @@
-include_set Abstract::WikirateTable
+include_set Abstract::WikirateTable # deprecated, but not clear if it is still used
```

Comment:
Soonish I'd like to make it so that `Metric+Company+*ltype rtype+*default` gives us a Record (type), and then all of this is in Record.

Biggish migration, but tiny code change.